templates/android.yml

```diff
@@ -18,3 +18,5 @@ steps:
     publishJUnitResults: false
     testResultsFiles: '**/TEST-*.xml'
     tasks: 'assembleDebug'
+
+- upload:
```

Comment:
Do we want a bunch of content-less `upload:` statements? I know it works, but would it be nicer to put a minimatch of `**/*` here so that it's obvious to users what's getting uploaded?

Reply:
We don't have filtered upload yet so ```**/*``` is misleading. But we could do ```- upload: $(System.DefaultWorkingDirectory)``` but that looked pretty ugly. On filtered upload - a minimatch pattern makes sense if you know what the root directory is (implied is the defautl working directory).

I'm going to start a side thread on this because I've been wrestling with this for a while so I want to get some input to help get my thinking straight.